networkx/algorithms/planarity.py

```diff
@@ -5,7 +5,8 @@
 
 
 def check_planarity(G, counterexample=False):
-    """Check if a graph is planar and return a counterexample or an embedding.
+    """Check if a graph is planar and return a Planar Embedding if true,
```

Comment:
Just a nit but I prefer the original wording - the first line of the docstring should IMO be as concise a summary as possible... details about the conditional return values can be fleshed out in the `Returns` section.